<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataEditorWidget</class>
 <widget class="QWidget" name="DataEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>æ•°æ®ç¼–è¾‘å™¨</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    /* å…¨å±€è®¾ç½®ï¼šæµ…è‰²èƒŒæ™¯ï¼Œé»‘è‰²å­—ä½“ */
    QWidget { background-color: #f5f6fa;
font-family: &quot;Microsoft YaHei&quot;, Arial; color: #000000; }

    /* æŒ‰é’®è®¾ç½® */
    QPushButton {
        background-color: #4a90e2;
color: white;
        border-radius: 4px;
        padding: 6px 12px;
        border: none;
        font-weight: bold;
    }
    QPushButton:hover { background-color: #357abd;
}
    QPushButton:disabled { background-color: #cccccc; color: #666666;
}

    /* è¾“å…¥æ¡†è®¾ç½® */
    QLineEdit {
        background-color: white;
border: 1px solid #ccc;
        border-radius: 4px;
        padding: 4px;
        color: black;
}

    /* è¡¨æ ¼è®¾ç½® */
    QTableView {
        background-color: white;
border: 1px solid #ddd;
        gridline-color: #eee;
        color: black;
        selection-background-color: #e0f0ff;
        selection-color: black;
}
    QHeaderView::section {
        background-color: #f8f9fa;
        border: none;
border-bottom: 1px solid #ddd;
        border-right: 1px solid #ddd;
        padding: 4px;
        color: #000000;
        font-weight: bold;
}

    /* æ ‡ç­¾è®¾ç½® */
    QLabel { color: #000000; background: transparent;
}

    /* åˆ†éš”çº¿è®¾ç½® */
    Line { background-color: #ccc;
}
   </string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <layout class="QHBoxLayout" name="topLayout">
     <item>
      <widget class="QPushButton" name="btnOpenFile">
       <property name="text">
        <string>ğŸ“ æ‰“å¼€æ–‡ä»¶</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>ğŸ’¾ ä¿å­˜</string>
       </property>
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>

     <item>
      <widget class="QPushButton" name="btnExport">
       <property name="text">
        <string>ğŸ“¤ å¯¼å‡º Excel</string>
       </property>
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>

     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDefineColumns">
       <property name="text">
        <string>ğŸ·ï¸ å®šä¹‰åˆ—</string>
       </property>
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnTimeConvert">
       <property name="text">
        <string>â° æ—¶é—´è½¬æ¢</string>
       </property>
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPressureDropCalc">
       <property name="text">
        <string>ğŸ“‰ å‹é™è®¡ç®—</string>
       </property>
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCalcPwf">
       <property name="text">
        <string>ğŸŒŠ æµå‹è®¡ç®—</string>
       </property>
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>

     <item>
      <widget class="QPushButton" name="btnErrorCheck">
       <property name="text">
        <string>âš ï¸ é”™è¯¯æ£€æŸ¥</string>
       </property>
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>

     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="searchLineEdit">
       <property name="placeholderText">
        <string>ğŸ” æœç´¢...</string>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="dataTableView">
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectionBehavior::SelectItems</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bottomLayout">
     <item>
      <widget class="QLabel" name="filePathLabel">
       <property name="text">
        <string>å½“å‰æ–‡ä»¶: æ— </string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="statusLabel">
       <property name="text">
        <string>å°±ç»ª</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
